src/olympia/reviewers/templatetags/assay.py

```diff
@@ -0,0 +1,12 @@
+from django import template
+
+
+register = template.Library()
+
+
+@register.simple_tag
+def assay_url(addon_guid, version, file=None):
```

Comment:
I think these args can be a less ambiguous (is version a Version object?) if it's a version string, `version_string` is clearer. Similarly, `file` isn't a File object, its a path and filename, so `path` or `filepath`?